r2dbc-core/src/main/java/io/micronaut/r2dbc/graalvm/R2dbcAutomaticFeature.java

```diff
@@ -35,15 +35,14 @@ public void beforeAnalysis(BeforeAnalysisAccess access) {
         Class<?> mariaDriver = access.findClassByName("org.mariadb.r2dbc.MariadbConnectionFactoryProvider");
         // handle mariadb
         if (mariaDriver != null) {
-
             AutomaticFeatureUtils.initializeAtRunTime(
                     access,
                     "org.mariadb.r2dbc.MariadbConnectionFactoryProvider",
                         "org.mariadb.r2dbc.client.ClientBase",
                         "org.mariadb.r2dbc.message.flow.AuthenticationFlow",
                         "org.mariadb.r2dbc.util.DefaultHostnameVerifier"
                     );
-
+            AutomaticFeatureUtils.addResourcePatterns("project.properties");
```

Comment:
can you explain this line?

Reply:
What do you want to explain? The change is needed for the native execution

Reply:
is `project.properties` a file specific for R2DBC projects?

Reply:
No, It's part of the internal driver impl